src/modules/networks/getBackendNetwork.ts

```diff
@@ -0,0 +1,44 @@
+import { ethers } from 'ethers';
+import { rejectAfterDelay } from 'src/shared/rejectAfterDelay';
+import { getNetworksBySearch } from '../ethereum/chains/requests';
+import { networksStore } from './networks-store.background';
+import type { Chain } from './Chain';
+import type { NetworkConfig } from './NetworkConfig';
+
+function maybeLocalChainId(id: string) {
+  return id.length === 21; // nanoid() standart length
```

Comment:
Not sure if it is safe to rely on the id length... Are there any other ways to check for local chain id/network?

Reply:
Yes :(